azerothcore/azerothcore-wotlk
# fix(DB/creature_template): Resistances for level 1-20 creatures

## Changes Proposed:
-  Align creature resistances for levels 1-20 with vmangos
-  Done after some discussions here https://github.com/azerothcore/azerothcore-wotlk/issues/4670 and https://github.com/chromiecraft/chromiecraft/issues/107


## Issues Addressed:
- Some resistances seem a bit high for low level creatures - no idea of the origin of these numbers, as it's hard to confirm



## Tests Performed:
- Looked at numbers to be changed, it could make sense
- Applied SQL
- Not tested in game



## How to Test the Changes:

- `.go c id <any entry id from the SQL>`
- Check combat log, use appropriate spells

## Known Issues and TODO List:

- The SQL looks like it does since I created it using code... Not super pretty, but the intention is showing in a fairly easy to read manner, exactly what will change.
- I prefer to segment these changes, if they are interesting, into ~10 level PR's going forward, not to swamp any reviewers with possibly hundreds of creatures at once


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
